example/etcd-client-svc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    instance: etcd-test
+    name: etcd
+  name: etcd-test-client
+  namespace: default
+spec:
+  clusterIP: 100.104.156.34
+  clusterIPs:
+    - 100.104.156.34
```

Comment:
```suggestion
```
Shall we remove this ClusterIP details

Reply:
this is not private data and can be any IP. If you are not convinced maybe we can change it to 192.168.*.* IPs